### ISSUE TITLE
SyntaxError for parser-flow on Node@4

### ISSUE THREAD

**Issue by cvbuelow** [prettier/prettier#1986]
When `parser-flow.js` is run through `uglify-es` the `parse` function is transformed to this (formatting added):
```js
function parse(t) {
  const r = flow_parser.parse(t, {
    esproposal_class_instance_fields: !0,
    esproposal_class_static_fields: !0,
    esproposal_export_star_as: !0 });
  if (r.errors.length > 0) {
    const t = r.errors[0].loc;
    throw createError(r.errors[0].message, {
      start: { line: t.start.line, column: t.start.column + 1 },
      end: { line: t.end.line, column: t.end.column + 1 } })
  }
  return includeShebang(t, r), r
}
```
When this is run on Node@4 the error `SyntaxError: Identifier 't' has already been declared` is thrown because `const t` is redeclaring `t` from the function argument.

**Comment by vjeux** [comment]
Thanks for raising this issue. Would you mind sending a PR that fixes the issue?

**Comment by vjeux** [comment]
I just published 1.4.3 which fixes it. Thanks for reporting it!
